### PR TITLE
Fix small bug and error

### PR DESCRIPTION
**Pull request by B0ZIA**

### CODE REVIEW

File: Objects/EventItem/EventManager.gd [hunk 1]
```diff
@@ -17,10 +18,12 @@ func wait_to_spawn_item():
 	timer.start()
 
 func return_random_time() -> float:
-	var value = randi()%40+20
+	var value = randi()%2+1#%40+20
```
Comment (Ajver) [inline Objects/EventItem/EventManager.gd]: I see You forgot to return previous value ;)

File: Scripts/GameSaver.gd [hunk 2]
```diff
@@ -14,6 +14,7 @@ onready var game_data = get_parent().get_node("GameData")
 onready var level_manager = get_parent().get_node("LevelManager")
 onready var elf_stats = get_parent().get_node("ElfStats")
 onready var elf = get_parent().get_node("Elf")
+onready var publician = get_parent().find_node("Publician")
```
Comment (Ajver) [inline Scripts/GameSaver.gd]: Little typo: should be "publican" without second 'i' (in find_node change node name as well)

File: Objects/EventItem/EventManager.gd (hunk 1 above)
Comment (B0ZIA) [reply Objects/EventItem/EventManager.gd]: Yes

File: Scripts/GameSaver.gd (hunk 2 above)
Comment (B0ZIA) [reply Scripts/GameSaver.gd]: Oh yes

**Pull request merged**